bsp/uefi/efi/Makefile

```diff
@@ -55,8 +55,15 @@ endif
 INCDIR := /usr/include
 
 # gnuefi sometimes installs these under a gnuefi/ directory, and sometimes not
-CRT0 := $(LIBDIR)/crt0-efi-$(ARCH).o
-LDSCRIPT := $(LIBDIR)/elf_$(ARCH)_efi.lds
+ifneq ("$(wildcard $(LIBDIR)/gnuefi/crt0-efi-$(ARCH).o)","")
```

Comment:
Nitpick, you could just make this `ifneq ($(wildcard $(LIBDIR)/gnuefi/*),)` (it makes it a little more readable).

Reply:
Ha.  OK.  I thought it was best to check for an expected file (for that $ARCH rather than just having the folder there.